README.md

```diff
@@ -36,3 +36,11 @@ Runs tests in console, in browser or with coverage.
 ### `lint`, `prettier`
 
 Runs linting and formatting for all files in `src` folder.
+
+### `npm run deploy`
+
+Add and configure S3 bucket creation and website deployment using AWS CDK and CloudFront Distribution using AWS CDK
+
+### `npm run destroy`
+
+CloudFront Invalidation using AWS CDK
```

Comment:
This command should destroy the stack, not invalidate paths

Reply:
I updated the code according to your comments